<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главная страница</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 239, 213)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="viewTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 224, 230)</string>
    </property>
    <property name="text">
     <string>Показать данные</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addInformayionInTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 224, 230)</string>
    </property>
    <property name="text">
     <string>Добавить данные в таблицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>640</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 224, 230)</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeInformationInTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 224, 230)</string>
    </property>
    <property name="text">
     <string>Удалить данные из таблицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateInformationInTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 224, 230)</string>
    </property>
    <property name="text">
     <string>Обноавить данные в таблице</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>460</width>
      <height>460</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>png/pngegg (1).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>831</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>33</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Аэропорт</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
